### ISSUE TITLE
Does the Preview button still have use?

### ISSUE THREAD

**Issue by d4mation** [joelworsham/cd-new-temp#3]
I remember you saying that the Preview button used to be necessary as auto-refreshing was not working yet. However, now that things auto-refresh (And fairly quickly), does it need to be there?

Or should it stay for the placebo effect of allowing the user to force-refresh the Preview even if it is 100ms away from doing it itself 😆 

If that's the case, maybe it should be something more immediately recognizable as a "Refresh" button?

![image](https://cloud.githubusercontent.com/assets/7770631/23512576/16aaaeec-ff2f-11e6-80a1-6151fbaf4c6c.png)

**Comment by d4mation** [comment]
Although I just realized you are using `fa-refresh` as a "Reset to Defaults" button, so I see now why you decided to use a different icon.

**Comment by joelworsham** [comment]
Yeah I was wondering the necessity of it myself. Part of me likes it because there's times I feel like I can't get the WP Customizer to refresh when I want it to and I have to figure out other ways to get it to. So, maybe it's unnecessary but I like it for the following reasons:

1) It's a good indicator of when the refreshing is happening, because it changes to a spinner.
2) It gives the user peace of mind. They can hit refresh themselves if they want and **know** it's refreshed.
3) If, for some reason, it won't auto-refresh, they can hit themselves.

That said, I am not opposed to ditching it. Hard call. Also, we can definitely change the reset role button to something else if need be.
